src/uptane/initialize.cc

```diff
@@ -218,8 +236,6 @@ InitRetCode Repository::initEcuRegister() {
 
 // Postcondition: "ECUs registered" flag set in the storage
 bool Repository::initialize() {
-  if (storage.loadEcuRegistered()) return true;
-
   // if device has aready been registered, just create new device and reprovision
   //   it should be changed/fixed when the backend supports registration acknowledgement
```

Comment:
@patrickvacek 
This comment really belongs to line 239 and propably needs to be rephased. I meant that if device with the same ID has been already registered to the server, aktualizr should generate a new ID and repeat registration. That is what this code hopefully does.